### PR TITLE
index updated pool metadata contract

### PR DESCRIPTION
**Pull request by devinaconley**
Indexing new `PoolMetadata` contract from https://github.com/gysr-io/aux/pull/3

### CODE REVIEW

File: subgraph.template.yaml [hunk 1]
```diff
@@ -186,6 +186,27 @@ dataSources:
...
+      startBlock: {{pool_metadata_v3_start_block}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Pool
+      abis:
+        - name: PoolMetadata
+          file: ./abis/PoolMetadata.json
```
Comment (holt-crews) [inline subgraph.template.yaml:204]: just double checking but the abi is the same for `PoolMetadata` and `PoolMetadataV3` right?

**Review by peters-josh — APPROVED** [review]
lgtm unless holt's comment is valid 🙂

File: subgraph.template.yaml (hunk 1 above)
Comment (devinaconley) [reply subgraph.template.yaml:204]: good question, yep same event definition (which is all we need from the ABI)

**Review by holt-crews — APPROVED** [review]
idk too much about this stuff but logically it makes sense to me

**Review by IainM22 — APPROVED** [review]

**Comment by devinaconley** [comment]
thanks all!

**Pull request merged**